Documentation/compression.md

```diff
@@ -0,0 +1,31 @@
+# Compression
+
+## Client side
+The preferred method for configuring message compression on a client is to pass `options` when the client object is instantiated.
+
+The two options need to be passed:
+
+**grpc.default_compression_algorithm** (integer)
+
+The option keeps the value of a compression algorithm.
```

Comment:
This should include a description of the functionality of this option. Based on the comments in [this header file](https://github.com/grpc/grpc/blob/master/include/grpc/impl/codegen/compression_types.h), I think it controls the default compression algorithm the channel will use to send messages.